jms/src/main/resources/reference.conf

```
@@ -53,6 +52,10 @@ alpakka.jms {
     # For use with transactions, if true the stream fails if Alpakka rolls back the transaction
     # when `ack-timeout` is hit.
     fail-stream-on-ack-timeout = false
+    # Max interval before sending queued acknowledges back to the broker. (Used by AckSources.)
+    # max-ack-interval = 5 seconds
+    # Max number of acks queued by AckSource before they are sent to broker. (Unless MaxAckInterval is specified).
+    max-pending-acks = 100
```

```suggestion
    max-pending-acks = ${alpakka.jms.consumer.buffer-size}
```

This will let the change be transparent to users who used `buffer-size` to control the pending acks in previous versions.

---

Nice